core/src/cgroup.rs

```diff
@@ -173,12 +174,15 @@ impl CGroup {
         fs::write(killfile, "1")?;
 
         // Check if all processes were terminated successfully
-        for i in 1..=64 {
-            trace!("kill iteration ({i}/64)");
+        let start = Instant::now();
+        trace!("Killing with a 1s timeout");
+        while start.elapsed() < Duration::from_secs(1) {
```

Comment:
maybe introduce a const variable for the maximum timeout ?

Reply:
sounds good

Reply:
done